### PR TITLE
New package: dislocker-0.7.1

### PR DESCRIPTION
**Pull request by samsky72**

### CODE REVIEW

**Comment by newbluemoon** [comment]
With `build_style=cmake` cmake is pulled in automatically, you don’t need to add it to hostmakedepends.

Checking `dislocker-0.7.1/src/CMakeLists.txt` you can specify the library directory with 

    configure_args="-DLIB_INSTALL_DIR=lib"

so you can remove `post_install()` and all should be fine. :)

**Comment by samsky72** [comment]
Thank you for your help, newbluemoon!

**Comment by newbluemoon** [comment]
@samsky72 No problem :)

But when making changes you should use `git commit --amend` and then force push so that there is just one commit per new/changed package in the PR. Or you can squash multiple commits into one via
`git rebase -i ...`.

File: srcpkgs/dislocker/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'dislocker'
+pkgname=dislocker
+version=0.7.1
+revision=1
+build_style=cmake
+configure_args="-DLIB_INSTALL_DIR=lib"
+makedepends="mbedtls-devel fuse-devel"
+depends="fuse mbedtls"
```
Comment (Vaelatern) [inline srcpkgs/dislocker/template]: This line should be deleted.
https://travis-ci.org/voidlinux/void-packages/jobs/323174884#L1053-L1058

Comment (samsky72) [reply srcpkgs/dislocker/template]: Ok, the line has been removed.

**Pull request closed without merging**